improving README.md (#36)
pubnub/arduino: README.md

has a class compatible with Arduino de facto standard `Client`. Your 
application can receive (subscribe to) and send (publish) messages.

## Full documentation

> Full detailed documentation of PubNub Arduino SDK can be found here:
>  [https://www.pubnub.com/docs/sdks/arduino](https://www.pubnub.com/docs/sdks/arduino

Reviewer: ```suggestion
For more information on Arduino, refer to the [PubNub Arduino SDK documentation](https://www.pubnub.com/docs/sdks/arduino).
```
